packages/wonder-blocks-testing/src/harness/types.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+import * as React from "react";
+
+/**
+ * A adapter to be composed with our test harnass infrastructure.
+ */
+export type Adapter<TConfig> = (
+    children: React.Node,
+    config: TConfig,
+) => React.Element<any>;
+
+/**
+ * A general map of adapters by their identifiers.
+ *
+ * It's OK that this has `any` for the config type as this is the very base
+ * version of a adapter set. In reality, a more specific type will be used
+ * with the harness functions that use more specific definitions of known
+ * adapters. This is just to support the base reality of not knowing.
+ *
+ * Use this on input positions only. Output positions for adapters
+ * should infer their type in most cases to ensure the strongest typing of
+ * the adapters.
+ */
+export type Adapters = {|
```

Comment:
Probably should rename this to `HarnessAdapters` to avoid confusion with the fixture framework.